### ISSUE TITLE
ResizeColumnsPlugin causes header tooltips to behave weird

### ISSUE THREAD

**Issue by jhickman** [DominoKit/domino-ui#732]
When using the ResizeColumnsPlugin (RC14), the tooltips on the headers behave strangely.

Sometimes the tooltip becomes.. "detached" from the header and doesn't realize that it should go away.   The easiest way to reproduce this is to resize a column past the right side so that it's hidden from overflow.   The tooltip of that column will remain persistent on the page.   It's not until you scroll to the right and mouse over the header before it disappears.

Another issue with tooltips and the ResizeColumnsPlugin is during the resize.  This seems to happen more on Chrome in Windows, but I have noticed that the tooltip will bounce between the two columns surrounding the resizing handle.   This is probably due to mouse drag speed and whatever header the mouse is over at the time; which can cause for some strobing effect.  

Possible solution:
Temporarily disable DataTable header tooltips during the drag of resizing a column.    

*Screenshot*
<img width="742" alt="image" src="https://user-images.githubusercontent.com/861908/204810233-11bf256b-7b18-4951-85ff-585ff12d3fca.png">
